R/prepare_tables.R

```diff
@@ -1,3 +1,22 @@
+#' Get patients for a selected time period and return as a list
+#' Beased on this list all the tables will be filtered
+#' 
+#' @param df the dataframe which patients selects from
+#' @param start_date begining of the period
+#' @param end_date end of the period
+#' 
+#' @return 
+#' @export
+#' 
+
+get_period_patients <- function(df, start_date, end_date){
+  start_date =  as.Date(start_date, format= '%d-%m-%Y')
```

Comment:
I recommend always asking users to enter dates in this format '2022-07-15', and always print them out in the same (unless it's for a graph).
This is the ISO standard, and makes sure there's never confusion about day first or month first.

Reply:
This is done! Please check